cotel/src/main/scala/day2/SpreadsheetChecksum.scala

```diff
@@ -0,0 +1,35 @@
+package day2
+
+
+object SpreadsheetChecksum {
+  type Row = List[Int]
+  type SpreadSheet = List[Row]
+
+  def Row(xs: Int*) = List(xs: _*)
```

Comment:
Int*  ??

xs: _* ???

Reply:
`Int*` is equivalent to `vararg numbers: Int` in Kotlin.

`xs: _*` is _type ascription_ which I am still reading about.